packages/groff.rb

```diff
@@ -3,21 +3,13 @@
 class Groff < Package
   description 'Groff (GNU troff) is a typesetting system that reads plain text mixed with formatting commands and produces formatted output.'
   homepage 'https://www.gnu.org/software/groff/'
-  version '1.22.3-1'
-  source_url 'http://ftp.gnu.org/gnu/groff/groff-1.22.3.tar.gz'
-  source_sha256 '3a48a9d6c97750bfbd535feeb5be0111db6406ddb7bb79fc680809cda6d828a5'
+  version '1.22.4'
+  source_url 'https://ftp.gnu.org/gnu/groff/groff-1.22.4.tar.gz'
```

Comment:
Can we change this to `source_url 'https://ftpmirror.gnu.org/groff/groff-1.22.4.tar.gz'`?